components/org.wso2.carbon.identity.api.server.image.service/org.wso2.carbon.identity.api.server.image.service.v1/src/gen/java/org/wso2/carbon/identity/api/server/image/service/v1/model/InlineObject.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Can we get rid of the inline object definitions, so that these objects will have proper names?

Reply:
fixed with 856301b8798d0f40eba305578984612be7a857f9